src/components/App/app-top-bar.tsx

```diff
@@ -0,0 +1,146 @@
+
+const AppTopBar: FunctionComponent = () => {
+    const dispatch = useDispatch();
+    const user = useSelector((state: AppState) => state.user);
+    const userManagerInstance = useSelector(
+        (state: AppState) => state.userManager?.instance
+    );
+
+    const navigate = useNavigate();
+    const matches = useMatches();
+    const selectedTabValue = useMemo(() => {
+        const handle: any = matches
+            .map((match) => match.handle)
+            .filter((handle: any) => !!handle?.appBar_tab)
+            .shift();
+        const tabValue: MainPaths = handle?.appBar_tab;
+        return tabValue && tabs.has(tabValue) ? tabValue : false;
+    }, [matches]);
+
+    const [appsAndUrls, setAppsAndUrls] = useState<
+        Awaited<ReturnType<typeof AppsMetadataSrv.fetchAppsAndUrls>>
+    >([]);
+
+    const theme = useSelector((state: AppState) => state[PARAM_THEME]);
+    const [themeLocal, handleChangeTheme] = useParameterState(PARAM_THEME);
+    const [languageLocal, handleChangeLanguage] =
+        useParameterState(PARAM_LANGUAGE);
+
+    const [showParameters, setShowParameters] = useState(false);
+
+    useEffect(() => {
+        if (user !== null) {
+            AppsMetadataSrv.fetchAppsAndUrls().then((res) => {
+                setAppsAndUrls(res);
+            });
+        }
+    }, [user]);
+
+    return (
+        <>
+            <TopBar
+                appName={APP_NAME}
+                appColor="#FD3745"
+                appLogo={
+                    theme === LIGHT_THEME ? (
+                        <GridAdminLogoLight />
+                    ) : (
+                        <GridAdminLogoDark />
+                    )
+                }
+                appVersion={AppPackage.version}
+                appLicense={AppPackage.license}
+                onParametersClick={() => setShowParameters(true)}
+                onLogoutClick={() => logout(dispatch, userManagerInstance)}
+                onLogoClick={() => navigate('/', { replace: true })}
+                user={user}
+                appsAndUrls={appsAndUrls}
+                globalVersionPromise={() =>
+                    AppsMetadataSrv.fetchVersion().then(
+                        (res) => res?.deployVersion
+                    )
+                }
+                additionalModulesPromise={StudySrv.getServersInfos}
+                onThemeClick={handleChangeTheme}
+                theme={themeLocal}
+                onLanguageClick={handleChangeLanguage}
+                language={languageLocal}
+            >
+                <Tabs
+                    component="nav"
+                    variant="scrollable"
+                    scrollButtons="auto"
+                    aria-label="Main navigation menu"
+                    sx={{ visibility: !user ? 'hidden' : undefined }}
+                    value={selectedTabValue}
+                >
+                    {[...tabs.values()]}
+                </Tabs>
+            </TopBar>
+            <Parameters
```

Comment:
In my opinion, we should not have parameters for now in Gridadmin, similarly to what is done in Gridexplore. parameters.tsx can be removed too.

Reply:
okay, will do in another pr to lighten this one